run-tests.sh

```diff
@@ -71,7 +71,7 @@ if [ $# -eq 0 ]
 
         set -e
         # remove this when https://github.com/pypa/pipenv/issues/4147 will be solved
-        pipenv check -i 37752
+        pipenv check -i 37752 -i 38076
```

Comment:
We dont need `-i 38076` it was solved with https://github.com/rero/rero-ils/pull/854